lib/index.js

```diff
@@ -34,22 +34,20 @@ const
      * @param {Array} element - All members of an Item or ItemGroup
      * @param {String} [path] - Path leading to members
      */
-    flattenEvents = function (element, path = []) {
+    flattenEvents = function (element = [], path = []) {
+        if (!(element.events && element.items)) { return {}; }
```

Comment:
This logic seems odd. What if event is present but item is missing, then we would want to add the events and skip the items